src/OmniSharp.Roslyn.CSharp/Services/Refactoring/V2/RunCodeActionService.cs

```diff
@@ -197,32 +160,57 @@ private async Task<IEnumerable<ModifiedFileResponse>> GetFileChangesAsync(Soluti
                 foreach (var documentId in projectChange.GetChangedDocuments())
                 {
                     var newDocument = newSolution.GetDocument(documentId);
+                    var oldDocument = oldSolution.GetDocument(documentId);
                     var filePath = newDocument.FilePath;
 
-                    if (!filePathToResponseMap.TryGetValue(filePath, out var modifiedFileResponse))
+                    // file rename
```

Comment:
`RenameDocumentOperation` doesn't exist in Roslyn 3.1 anymore, renames are normal `ApplyChangesOperation` so I needed to update this code a bit to make this work. On the upside, there is no reflection anymore